tools/sof-kernel-log-check.sh

```diff
@@ -381,7 +381,7 @@ ignore_str="$ignore_str"'|I/O error, dev loop., sector 0 op 0x0:.READ. flags 0x8
 
 # NVME harmless errors added in 5.18-rc1
 # https://github.com/thesofproject/sof-test/issues/888
-ignore_str="$ignore_str"'|nvme0: Admin Cmd(0x[:digit:]+), I/O Error (sct 0x0 / sc 0x2)'
+ignore_str="$ignore_str"'|nvme0: Admin Cmd(0x[[:digit:]+]), I/O Error (sct 0x0 / sc 0x2)'
```

Comment:
I think you mean `Admin Cmd(0x[[:digit:]]+)` ?

Test it with `echo Cmd(0x1234)` | grep -E ...`

Reply:
yes, done